core/src/main/java/com/exadel/aem/backpack/core/services/impl/PackageServiceImpl.java

```diff
@@ -304,29 +318,150 @@ public PackageInfo createPackage(final ResourceResolver resourceResolver, final
     }
 
     /**
-     * Called by {@link PackageServiceImpl#createPackage(ResourceResolver, CreatePackageModel)} to implement package
+     * {@inheritDoc}
+     */
```

Comment:
(very cosmetic) If I'm not mistaken, javadocs are inherited by default for overridden methods and it makes sense to use inheritDoc only if you want to add something to the superclass javalog

Reply:
Actually, sometimes {@inheritDoc} is used for mere visual control, like "been here" or  "Do I have all my methods javadoc-covered? Is there javadoc in my superclass?". So, it's OK for me)